addons/rating_project/views/project_view.xml

```diff
@@ -32,7 +36,7 @@
             <field name="inherit_id" ref="project.view_task_form2"/>
             <field name="arch" type="xml">
                 <xpath expr="//button[@name='toggle_active']" position="before">
-                    <button name="%(rating_rating_action_task)d" type="action"
+                    <button name="action_open_rating_view" type="object"
```

Comment:
Why this change ? Can't you keep action defined in xml (and eventually modify it) ?